<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="ImageViewer" name="imageview" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_open">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>open video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calibration">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>calibration</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>FPS</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>211</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>135</width>
        <height>129</height>
       </rect>
      </property>
      <widget class="QCheckBox" name="box_check">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>face box</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="landmark_check">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>face landmarks</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>eye track</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>191</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QWidget</extends>
   <header location="global">imageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
